tests/test_regex_identifier.py

```diff
@@ -475,6 +478,36 @@ def test_arn4():
     res = r.check(["arn:aws:s3:::my_corporate_bucket/Development/*"])
     assert "ARN" in str(res)
 
+
+def test_unix_timestamp():
+    r = regex_identifier.RegexIdentifier()
+
+    res = r.check(["1577836800"])  # 2020-01-01
+    keys = [m['Regex Pattern']['Name'] for m in res]
+    assert "Unix Timestamp" in keys
+    assert "Recent Unix Timestamp" in keys
+
+    res = r.check(["94694400"])  # 1973-01-01
+    keys = [m['Regex Pattern']['Name'] for m in res]
+    assert "Unix Timestamp" in keys
+    assert "Recent Unix Timestamp" not in keys
+
+    res = r.check(["1234567"])  # 7 numbers
+    keys = [m['Regex Pattern']['Name'] for m in res]
+    assert "Unix Timestamp" not in keys
+    assert "Recent Unix Timestamp" not in keys
+
+    res = r.check(["1577836800000"])  # 2020-01-01
+    keys = [m['Regex Pattern']['Name'] for m in res]
+    assert "Unix Millisecond Timestamp" in keys
+    assert "Recent Unix Millisecond Timestamp" in keys
+
+    res = r.check(["94694400000"])  # 1973-01-01
+    keys = [m['Regex Pattern']['Name'] for m in res]
+    assert "Unix Millisecond Timestamp" in keys
+    assert "Recent Unix Millisecond Timestamp" not in keys
```

Comment:
Can you separate groupse to be in different test functions? Like `test_unix_timestamp2()` and so on

Reply:
separated 👌